nixos/modules/misc/ids.nix

```diff
@@ -338,6 +338,7 @@
       minetest = 311;
       rss2email = 312;
       cockroachdb = 313;
+      thelounge = 314;
```

Comment:
As this service only will allocate a few files, then become dynamic ids as well.